src/js/common/messageHelper.js

```diff
@@ -35,7 +41,7 @@ fluid.defaults("gpii.handlebars.helper.messageHelper", {
     invokers: {
         "getHelper": {
             "funcName": "gpii.handlebars.helper.messageHelper.getHelper",
-            "args":     ["{that}"]
+            "args":     ["{that}", "{gpii.handlebars.renderer.serverAware}"]
```

Comment:
If we do end up needing both "server aware" and "non server aware" in the short term, I'm wondering if we can automatically handle this using context awareness so that people only need to use the single grade.